frc_characterization/newproject/__init__.py

```diff
@@ -478,7 +478,7 @@ def isRotation(units):
         )
 
         controlMenu = OptionMenu(
-            topFrame, self.control_type, *["Simple", "CTRE", "SparkMax"]
+            topFrame, self.control_type, *["Simple", "CTRE", "SparkMax", "PWF"]
```

Comment:
Change this to `Venom` or maybe `CANVenom` to make it a bit more clear what the option is for.
```suggestion
            topFrame, self.control_type, *["Simple", "CTRE", "SparkMax", "Venom"]
```

Reply:
Agreed